setup.py

```diff
@@ -43,4 +48,13 @@
         'Topic :: Documentation',
         'Topic :: Software Development :: Documentation',
     ],
+    install_requires= [
+        'Sphinx>=1.4.1',
+    ],
+    cmdclass = {
+        'compile_catalog': babel.compile_catalog,
```

Comment:
@mitya57 it is the problem :( 

then remove the Babel Distutils/setuptools integration? I had to install Babel via pip install so I could extract the strings to translate and compile the mo file.